packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -1547,6 +1570,63 @@ addFieldsDict(Users, {
     canRead: ['guests'],
   },
   
+  // How others can help me (Markdown version)
```

Comment:
I decided to make these two new text fields work the same way as the Bio field, since I thought that would be easier to understand for users.

Reply:
The bio field is tech debt. I would recommend building this better. If that makes it clash with the bio field... it might be time to upgrade the bio field. Or just let it clash with the bio field.